KomodoPlatform/komodo-wallet-desktop
# Fix zcash params download

Closes https://github.com/KomodoPlatform/komodo-wallet-desktop/issues/2331
To test:
- delete your local `zcash-params` folder. Location for each OS is here https://forum.komodoplatform.com/t/installing-zcash-params/603
- launch app, then login
- try to activate pirate or zombie
- you should see the modal below
- download params and confirm ARRR/ZOMBIE activates as expected.
![image](https://github.com/KomodoPlatform/komodo-wallet-desktop/assets/35845239/92931552-2ff7-4d30-81d0-4147b356e5be)
